launch_ros/launch_ros/actions/ros_timer_action.py

```diff
@@ -0,0 +1,220 @@
+# Copyright 2021 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Module for the RosTimerAction action."""
+
+import asyncio
+import collections.abc
+from functools import partial
+from typing import Any  # noqa: F401
+from typing import cast
+from typing import Dict  # noqa: F401
+from typing import Iterable
+from typing import List
+from typing import Optional
+from typing import Text
+from typing import Tuple
+from typing import Union
+
+from launch_ros.ros_adapters import get_ros_node
+
+import launch.logging
+from launch.actions.opaque_function import OpaqueFunction
+from launch.action import Action
+from launch.event_handler import EventHandler
+from launch.events import Shutdown
+from launch.events import TimerEvent
+from launch.frontend import Entity
+from launch.frontend import expose_action
+from launch.frontend import Parser
+from launch.launch_context import LaunchContext
+from launch.launch_description_entity import LaunchDescriptionEntity
+from launch.some_actions_type import SomeActionsType
+from launch.some_substitutions_type import SomeSubstitutionsType
+from launch.some_substitutions_type import SomeSubstitutionsType_types_tuple
+from launch.utilities import create_future
+from launch.utilities import ensure_argument_type
+from launch.utilities import is_a_subclass
+from launch.utilities import type_utils
+
+from rclpy.parameter import Parameter
+
+
+@expose_action('ros_timer_action')
```

Comment:
```suggestion
@expose_action('ros_timer')
```

We typically don't add an `_action` in those names

Reply:
I've changed the name from RosTimerAction to RosTimer.